cradle-core/src/main/java/com/exactpro/cradle/CradleStorage.java

```diff
@@ -311,9 +311,9 @@ public BookInfo refreshPages(BookId bookId) throws CradleStorageException, IOExc
 	/**
 	 * Getting information about books from storage and put it in internal cache
 	 * @return Collection of loaded books
-	 * @throws IOException if books data reading failed
+	 * @throws CradleStorageException if books data reading failed
 	 */
-	public Collection<BookInfo> refreshBooks() throws IOException
+	public Collection<BookInfo> refreshBooks() throws CradleStorageException
```

Comment:
Error while reading data still can occur, so `IOException` should be still possible to throw.

Reply:
Why we need to expose IOException for API consumers as for them it is basically CradleStorageException?

Reply:
Up!

Reply:
This is disputable, of course. My understanding is `IOException` will indicate that data reading failed while `CradleStorageException` indicates that some verifications didn't pass. I'm not insisting, but this is how this is done for the rest of the API.
Basically, `IOException` can be used as `cause` of `CradleStorageException`.

Reply:
Yep, enclosing IOException in cause is a good idea